external/fv3fit/fv3fit/keras/_models/dense.py

```diff
@@ -48,8 +48,6 @@ class DenseHyperparameters(Hyperparameters):
             variable names and values are either a scalar referring to the
             total weight of the variable. Default is a total weight of 1
             for each variable.
-        normalize_loss: if True (default), normalize outputs by their standard
```

Comment:
This was unused, i.e. it was not possible to compute loss on un-normalized outputs.